### PR TITLE
Hide Geometry Property

### PR DESCRIPTION
**Pull request by mdelez**
closes https://dasch.myjetbrains.com/youtrack/issue/DSP-649

uses dsp-js rc.11 and dsp-api rc.15

### CODE REVIEW

**Review by tobiasschweizer — APPROVED** [review]
neat

File: projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts [hunk 1]
```diff
@@ -135,7 +135,10 @@ export class ResourceViewComponent implements OnInit, OnChanges, OnDestroy {
                 );
 
                 // sort properties by guiOrder
-                this.resPropInfoVals.sort((a, b) => (a.guiDef.guiOrder > b.guiDef.guiOrder) ? 1 : -1);
+                this.resPropInfoVals =
+                    this.resPropInfoVals
+                        .filter(prop => prop.propDef.objectType !== 'http://api.knora.org/ontology/knora-api/v2#GeomValue')
```
Comment (tobiasschweizer) [inline projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts:140]: use a constant

**Review by tobiasschweizer — CHANGES_REQUESTED** [review]

File: projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts (hunk 1 above)
Comment (mdelez) [reply projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts]: constant added in 49f6602

**Review by tobiasschweizer — APPROVED** [review]
neater

**Comment by mdelez** [comment]
> neater

[That's pretty neat](https://www.youtube.com/watch?v=Hm3JodBR-vs)

**Pull request merged**